lib/Page.js

```diff
@@ -60,6 +60,7 @@ function Page(pageConfig) {
   this.frontMatter = {};
   this.headings = {};
   this.includedFiles = {};
+  this.depthOfHeadings = pageConfig.depthOfHeadings
```

Comment:
Missing semi-colon at the end. Do check the coding style by running `npm run lintwin` frequently. Travis is failing as well due to this issue.